src/main/java/dev/emi/trinkets/api/Trinket.java

```diff
@@ -107,6 +107,16 @@ default boolean canUnequip(ItemStack stack, SlotReference slot, LivingEntity ent
 		return map;
 	}
 
+	/**
+	 * Returns the Slot Modifiers for a stack in a slot. Child implementations should remain pure.
+	 * Keys should be in the format "group:slot".
+	 *
+	 * @param uuid The UUID to use for creating attributes
+	 */
+	default Multimap<String, EntityAttributeModifier> getSlotModifiers(ItemStack stack, SlotReference slot, LivingEntity entity, UUID uuid) {
```

Comment:
Do you think it might be beneficial to merge this and the normal modifier method and simply split them apart on the api side so it seems consistent?

Reply:
Hmm, that could be a good idea. I just don't know what that would look like since the maps have different key types. Would the combined method return a `Pair` of multimaps or perhaps utilize some new wrapper class? I guess it'd also be possible to convert attributes to some kind of string-key schema as well. In any case, I do think there's merit in attempting to merge them for the sake of streamlining.

Reply:
Aha, that's the reason, if we go with my idea of creating a builder method, it could construct the key itself and the user would be none the wiser